src/include/map.h

```diff
@@ -129,6 +129,9 @@ class CMapInfo
 
 	bool IsPointOnMap(const Vec2i &pos) const { return IsPointOnMap(pos.x, pos.y); }
 
+	bool IsHighgroundsEnabled() const { return Highgrounds; }
+	void EnableHighgrounds(const bool enable = true) { Highgrounds = enable; }
```

Comment:
```suggestion
	void EnableHighgrounds(bool enable = true) { Highgrounds = enable; }
```
(non-ref) parameters don't need to be `const`.

Reply:
Fixed https://github.com/Wargus/stratagus/pull/581/commits/f6231bfe97438217fbac7e2435577fb5904a61a3